02week/nasa/style.css

```diff
@@ -0,0 +1,80 @@
+img {
```

Comment:
Same thing here as with the resume css file, everything is indented two more spaces unnecessarily.